web/scripts/build-locales.mjs

```diff
@@ -27,17 +34,14 @@ function compareXlfAndSrc(loc) {
         return false;
     }
 
-    // if the xlf is newer (greater) than src, it's out of date.
-    if (xlfStat.mtimeMs > srcStat.mtimeMs) {
-        return false;
-    }
-    return true;
+    // if the xliff file is older (has a lower build time) than the generated translation file, the
+    // translation file is up-to-date
+    return xlfStat.mtimeMs <= srcStat.mtimeMs;
```

Comment:
This was what ESLint whined about.  "Just return the results of the last comparison!"